Lib/fontParts/base/point.py

```diff
@@ -40,7 +43,7 @@ class BasePoint(
         "name"
```

Comment:
`copyAttributes` should be `Tuple[str, str, str, str, str]`.

Reply:
Again, shouldn't the type checker be able to deduce it from the assignment statement?
Same issue with all the dynamic properties. I am used to work with Pyright, so that's maybe a specific feature.

Anyway, I'll add it.

Reply:
I think it depends on how the variables are used. For clarity, I always annotate global variables, while only including them for locals whenever `mypy` complains.